catalog/spatial/registry/registry-admin-modules/registry-admin-poller-service/src/test/java/org/codice/ddf/registry/admin/poller/AdminPollerTest.java

```diff
@@ -68,18 +70,28 @@
 
 public class AdminPollerTest {
 
-    public static final String CONFIG_PID = "properPid";
+    private static final String CONFIG_PID = "properPid";
 
-    public static final String EXCEPTION_PID = "throwsAnException";
+    private static final String EXCEPTION_PID = "throwsAnException";
 
-    public static final String FPID = "OpenSearchSource";
+    private static final String SOURCE_FPID = "OpenSearchSource";
 
-    public static MockedAdminPoller poller;
+    private static MockedSourceAdminPoller sourcePoller;
+
+    private static MockedRegistryAdminPoller registryPoller;
 
     private static final String REGISTRY_ID = "registry-id";
 
     private static final String PUBLISHED_LOCATIONS = "published-locations";
```

Comment:
should be able to replace with constant from registry metacard type for this and registry-id